Jenkinsfile

```diff
@@ -407,10 +409,47 @@ EOF
                 }
             }
         }
+        stage('Determine non-trigger files') {
+            steps {
+                script {
+                    def changesetFile = "non-trigger-files.txt"
```

Comment:
`non-trigger-files` is not very clear IMO. how about make it hidden and name it `.testignore` or `.e2eignore`?